eservice/pdo/eservice/enclave/enclave/base.cpp

```diff
@@ -140,8 +140,8 @@ pdo_err_t pdo::enclave_api::base::GetEpidGroup(
 
      try {
         // Get the EPID group from the enclave and convert it to big endian
-        sgx_epid_group_id_t epidGroup;
-        g_Enclave.GetEpidGroup(epidGroup);
+        sgx_epid_group_id_t epidGroup = { 0 };
+        g_Enclave.GetEpidGroup(&epidGroup);
 
```

Comment:
so... does this require the struct or a pointer? the copy below treats this as a pointer. could you verify the appropriate use of this structure.

Reply:
I've fixed the inconsistencies. The structure is declare, but after that we use pointer to that structure.